docs/mp/integrations/wls.adoc

```diff
@@ -0,0 +1,392 @@
+///////////////////////////////////////////////////////////////////////////////
+
+    Copyright (c) 2023 Oracle and/or its affiliates.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+///////////////////////////////////////////////////////////////////////////////
+
+
+= Oracle WebLogic Server 
+:description: Helidon Oracle WebLogic Server Integration
+:keywords: wls, jms, rest, single sign-on, sso, microtx, web services
+:feature-name: Oracle WebLogic Server Integration Integration
+:rootdir: {docdir}/../..
+
+include::{rootdir}/includes/mp.adoc[]
+
+== Contents
+
+- <<Overview, Overview>>
+- <<REST Services Integration, REST Services Integration>>
+- <<JMS Integration, JMS Integration>>
+- <<Web Services Integration, Web Services Integration>>
+- <<Single Sign-On on OCI Integration, Single Sign-On on OCI Integration>>
+
+== Overview
+
+The Oracle WebLogic Server and Helidon integration enables interaction between a Helidon microservice application and an application installed on Oracle WebLogic Server, which is deployed in a Kubernetes cluster managed by the WebLogic Kubernetes Operator (WKO).
```

Comment:
This statement is too restrictive. Kubernetes/Kubernetes Operator, etc. are not always required. I believe the language in the WLS docs says: "This document provides information about the Oracle WebLogic Server and Helidon integration when both products are deployed in a Kubernetes environment. **However, you can implement this integration on any supported platform**." (I added the bold text which may be sufficient)

Reply:
I will update this statement.